eth-riscv-syscalls/src/lib.rs

```diff
@@ -60,4 +62,6 @@ syscalls!(
     (4, Revert, "revert"),
     (5, Caller, "caller"),
     (0x20, Keccak256, "keccak256"),
+    (0x3A, GasPrice, "gasprice"),
+    (0x32, Origin, "origin"),
```

Comment:
```suggestion
    (0x32, Origin, "origin"),
    (0x3A, GasPrice, "gasprice"),
```
just to keep sorted by opcode